docs2/site/docs/getting-started/schema-types.md

```diff
@@ -112,21 +163,92 @@ public class EpisodeEnum : EnumerationGraphType
 }
 ```
 
-Or you can use the generic version passing it a .NET `enum` which will populate values and descriptions (if any defined via `DescriptionAttribute`) for you. The `Name` will default to the .NET Type name, which you can override in the constructor.
+Note that although GraphQL has no use for backing values for enum members, GraphQL.NET uses them anyway. This allows for a more natural mapping to .NET `enum`s or other collections of constants, and avoids coupling your business logic to GraphQL semantics. The backing values are strictly for use on the back end - the client will never see them.
+
+**Resolving Enumerations**
+
+Fields typed as enumerations are resolved by returning either the name or backing value of one or more of the enum members. In the below examples, notice the identical implementations of the `appearsIn` field for both human graph types. In both implementations, the client receives the enum member names in response to queries on the `appearsIn` field.
```

Comment:
> of one or more 

What do you mean?

Reply:
Clarified this in #1205775f. I meant to show that resolving enums works equally well with list types.

Reply:
OK